packages/target-electron/static/webxdc-preload.js

```diff
@@ -247,12 +247,14 @@ class RealtimeListener {
   const connections = []
 
   contextBridge.exposeInMainWorld('webxdc_internal', {
-    setup: (selfAddr, selfName) => {
+    setup: (selfAddr, selfName, sendUpdateInterval, sendUpdateMaxSize) => {
       if (is_ready) {
         return
       }
       api.selfAddr = Buffer.from(selfAddr, 'base64').toString('utf-8')
       api.selfName = Buffer.from(selfName, 'base64').toString('utf-8')
+      api.sendUpdateInterval = sendUpdateInterval ? sendUpdateInterval : 1000
+      api.sendUpdateMaxSize = sendUpdateMaxSize ? sendUpdateMaxSize : 18874368
```

Comment:
This are the values core is providing right now. Just used them as a reasonable fallback in case these values are empty for whatever reason. 
I just see that [here](https://c.delta.chat/classdc__msg__t.html#aee526949983962ed25adf33a2f5e6878) a different values recommended.  Anyway it's just a fallback
> And is it possible that sendUpdateInterval is nullish

Not AFAIK

Reply:
I'd leave a comment then stating that this should not happen.
Because I guess we wouldn't want to provide values that aren't actually the true values.

Reply:
i think as well, it is fine as is

there is no case where these values are unset in case core's or cffi's `get_webxdc_info()` return successfully.

not sure about jsonrpc - but as that probably just calls  core's `get_webxdc_info()` i assume this is not possible as well.

still, probably good to have some default (which could also be `undefined`, [though](https://github.com/webxdc/website/pull/94/files#diff-08cf571a0d83f52c2a62868c9fafed34f35bac7e5f72b73da3c52bc6c898fd02R83), but not `0` or `-1` or so)

Reply:
Adding an extra fallback is not needed as https://js.jsonrpc.delta.chat/types/T.WebxdcMessageInfo.html shows that it can not be `undefined` or `null`.